api/yelp_beans/logic/meeting_spec.py

```diff
@@ -41,7 +41,7 @@ def get_users_from_spec(meeting_spec):
                     preference_dt.weekday() == meeting_spec.datetime.weekday():
 
                 user = User.query.filter(
-                    User.id == user_preference.user_id).one()
+                    User.id == user_preference.user_id).first()
                 logging.info('user added: ')
                 logging.info(user)
                 users.append(user)
```

Comment:
Should we only add a user to the list of users if the query doesn't return None. Looks like [the couple of places that we call this](https://sourcegraph.com/search?q=context:global+repo:%5Egithub.com/Yelp/beans%24+get_users_from_spec%28&patternType=regexp) we already filter out None, but seems better to filter it out here. This changes the existing behavior, so feel free to drop this.

```suggestion
                    User.id == user_preference.user_id).first()
                if user is not None:
                    logging.info('user added: ')
                    logging.info(user)
                    users.append(user)
```